lobby/src/test/java/org/triplea/lobby/server/db/BadWordControllerIntegrationTest.java

```diff
@@ -5,13 +5,15 @@
 
 import com.github.database.rider.core.api.dataset.DataSet;
 import com.github.database.rider.junit5.DBUnitExtension;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.ValueSource;
 import org.triplea.lobby.server.config.TestLobbyConfigurations;
 import org.triplea.test.common.Integration;
 
+@Disabled
```

Comment:
Is it worth adding a project comment here?

Reply:
Same with the other tests

Reply:
It should not be disabled actually. Our `verify` script broke some time ago, this update really made that bad and I could no longer ever get clean runs as there was some conflict between one test or another. This was disabled as a way to try and get around that, eventually I submitted this expecting to see the tests conflict with one another and was a bit surprised they did not.
Removed the disabled: 2f062fb53